src/views/Lottery/components/Hero.tsx

```diff
@@ -60,26 +81,51 @@ const RightWrapper = styled.div`
 
   ${({ theme }) => theme.mediaQueries.lg} {
     padding-left: 32px;
+    & img {
+      margin-top: -25px;
+    }
   }
 `
 
 const Hero = () => {
   const TranslateString = useI18n()
+  const { isXs, isSm } = useMatchBreakpoints()
+
+  const text = (
+    <LeftWrapper>
+      <Title>{TranslateString(999, 'The Lottery Is Changing!')}</Title>
+      <ComeBack>{TranslateString(999, 'Come back soon!')}</ComeBack>
+      <LearnMore>
+        <LearnMoreLink href="https://voting.pancakeswap.finance/#/pancake/proposal/QmU8pcbmBrfbfVQXMMxmkExDq3mYq4s5cbBuFe6uCZzdmX">
```

Comment:
I think you can use the uikit `<Link>` with the `external` prop here, it'll also nicely handle some of the styles you need.

```
<Link bold  mt="20" external href={`https://google.com`}>
```